internal/handshake/hkdf.go

```diff
@@ -18,10 +18,9 @@ func hkdfExpandLabel(hash crypto.Hash, secret, context []byte, label string, len
 	b[3+6+len(label)] = uint8(len(context))
 	b = append(b, context...)
 
-	out := make([]byte, length)
-	n, err := hkdf.Expand(hash.New, secret, b).Read(out)
-	if err != nil || n != length {
-		panic("quic: HKDF-Expand-Label invocation failed unexpectedly")
+	expanded, err := hkdf.Expand(hash.New, secret, string(b), length)
+	if err != nil {
+		panic(fmt.Errorf("quic: HKDF-Expand-Label invocation failed unexpectedly: %v", err))
```

Comment:
does it help not adding fmt.errorf here and keep the static string?

right now upstream golang can return only the one error, and only in fips code path; hence a static panic message like before is still good enough for now. One can always get the full err when debugging.